libs/openchallenges/home/src/lib/challenge-search/challenge-search.component.ts

```diff
@@ -81,6 +84,12 @@ export class ChallengeSearchComponent implements OnInit {
         pageSize: 1,
       })
       .pipe(map((page) => page.totalElements));
+    this.platformCount$ = this.challengePlatformService
+      .listChallengePlatforms({
+        pageNumber: 1,
+        pageSize: 1,
```

Comment:
I just checked and we do require a page to have at least 1 item. Maybe we could set the min to 0 for the sake of making this query lighter (future work).